spyderlib/widgets/file_switcher.py

```diff
@@ -0,0 +1,405 @@
+# -*- coding: utf-8 -*-
+#
+# Augmented file switcher by DM, 2015 (based on original in widgets/editor.py)
+# Licensed under the terms of the MIT License
+# (see spyderlib/__init__.py for details)
+
```

Comment:
Do we have a standard way right now to handle this file headers, and authorship? I myself have doubts when creating new stuff from scratch. 

Should it be something like this:

```
# -*- coding: utf-8 -*-
#
# Copyright © 2013-2015 The Spyder Development Team
# Copyright © 2015 Daniel Manson (@d1manson)
# Licensed under the terms of the MIT License
# (see spyderlib/__init__.py for details)
```

or just 

```
# -*- coding: utf-8 -*-
#
# Copyright © 2013-2015 The Spyder Development Team
# Licensed under the terms of the MIT License
# (see spyderlib/__init__.py for details)
```

And an entry of the author in the [AUTHORS](https://github.com/spyder-ide/spyder/blob/master/AUTHORS) file?